docs/hardfork-activation-inheritance.md

```diff
@@ -15,9 +15,12 @@ At the time of writing, this is implemented for
 * [op-geth](https://docs.optimism.io/builders/node-operators/configuration/base-config#initialization-via-network-flags)
 * [op-node](https://docs.optimism.io/builders/node-operators/configuration/base-config#configuring-op-node)
 
-These components load configuration from the superchain registry. This implies some more conditions which need to hold for a chain to receive the superchain-wide hardfork activation:
+> [!NOTE]
+> Since version 2.0 of the superchain registry, the inherited hardfork activation times are applied directly to each chain's individual chain config TOML file. What you see is what you get (unlike in previous iterations of the registry, where the inheritance was applied "magically" by Go bindings).
```

Comment:
```suggestion
> Since [version 2.0](https://github.com/ethereum-optimism/superchain-registry/pull/816) of the superchain registry, the inherited hardfork activation times are applied directly to each chain's individual chain config TOML file. What you see is what you get (unlike in previous iterations of the registry, where the inheritance was applied "magically" by Go bindings).
```

Reply:
I'll link to the PR here, from where all of that information is discoverable.